view/view_manager.py

```
@@ -0,0 +1,38 @@
+
+class ViewManager:
+
+    def __init__(self):
+        self.main_window: Window = None
+        self.menu_frame: Menu = None
+        self.view_frame: ViewFrame = None
+
+        self.__init_main_window()
+        self.__init_menu_frame()
+
+    def __init_main_window(self) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)